### DISCUSSION TITLE
명예의 전당 관련 문의 드립니다.

### DISCUSSION THREAD

**Discussion by subeenpark-io** [devrel-kr/HackaLearn#418] [1 Q&A - HackaLearn 운영] [answered]
안녕하세요! 저희 팀(월급두배받는법)의 경우 6개 챌린지 모두 완료되었지만 명예의 전당에서 확인되지 않는데, 그냥 기다리면 될까요?
항상 열정적인 행사 운영 감사합니다.👍

**Comment by Eomhyunjun** [comment, marked as answer]
올라가셨어요! 정말 이쁘게 잘 만드셨네요!

**Reply by subeenpark-io** [reply]
엇 이제 확인했어요! 감사합니다 ㅎㅎ
